<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sam-PC" ignored="0" name="Default test" tests="5" failures="1" timestamp="18 May 2021 01:27:32 GMT" time="41.582" errors="0">
  <testcase name="loginTest" time="14.057" classname="com.crm.qa.testcases.LoginPageTest"/>
  <testcase name="signupbtnTest" time="0.147" classname="com.crm.qa.testcases.LoginPageTest"/>
  <testcase name="loginPageTitleTest" time="0.068" classname="com.crm.qa.testcases.LoginPageTest"/>
  <testcase name="crmlogoTest" time="0.118" classname="com.crm.qa.testcases.LoginPageTest"/>
  <testcase name="loginPagewrongTitleTest" time="0.064" classname="com.crm.qa.testcases.LoginPageTest">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[CRMPRO - CRM software for customer relationship management, sales, and support.]&amp;gt; but was:&amp;lt;[I want this test case to fail]&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[CRMPRO - CRM software for customer relationship management, sales, and support.]> but was:<[I want this test case to fail]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.crm.qa.testcases.LoginPageTest.loginPagewrongTitleTest(LoginPageTest.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- loginPagewrongTitleTest -->
</testsuite> <!-- Default test -->
